index.js

```diff
@@ -87,7 +87,9 @@ async function run() {
     await eventsRelay(
       params.txServerPort,
       bridgeState.web3,
-      bridgeState.contract
+      bridgeState.bridgeContract,
+      bridgeState.exitHandlerContract,
```

Comment:
Btw, `exitHandlerContract` doesn't seem like a meaningful name. `exitHandlerContract.methods.nftTokenCount().call()` doesn't make any sense

Reply:
Agreed...

Reply:
Resolved in latest commit.